src/frontends/src/bench.cpp

```diff
@@ -70,7 +70,6 @@ void send_predictions(std::unordered_map<std::string, std::string> &config, Quer
     std::vector<boost::future<Response>> futures;
     std::vector<int> binary_labels;
     for (int i = 0; i < batch_size; i++) {
-      std::srand(time(NULL));
```

Comment:
Seeding at every iteration of the loop caused `rand()` to consistently produce the same value. We now seed once prior to the execution of `send_predictions`